Cargo.toml

```diff
@@ -12,6 +12,7 @@ categories = ["api-bindings", "development-tools::ffi"]
 license = "Apache-2.0"
 exclude = ["/.gitignore", ".cargo/config", "/codecov.yml", "/Makefile", "/pyproject.toml", "/tox.ini"]
 edition = "2018"
+links = "python"
```

Comment:
Yeah I was wondering about this in the past too.

It's also good because it prevents users from accidentally trying to depend on different PyO3 versions, so I think this is 👍.

I guess it is technically breaking, so probably needs to go in 0.15